docs/speech-usage.rst

```diff
@@ -38,17 +38,18 @@ and returns alternative text transcriptons.
 
   .. code-block:: python
 
-     >>> alternatives = client.sync_recognize(None,"gs://my-bucket/recording.flac",
-     ...                 "FLAC", 16000, max_alternatives=2):
+     >>> alternatives = client.sync_recognize(
+     ...     None, 'gs://my-bucket/recording.flac',
+     ...     'FLAC', 16000, max_alternatives=2)
      >>> for alternative in alternatives:
      ...     print('=' * 20)
-     ...     print('   transcript: %s' % (alternative["transcript"],))
-     ...     print('   confidence: %s' % (alternative["confidence"],))
+     ...     print('transcript: ' + alternative['transcript'])
+     ...     print('confidence: ' + alternative['confidence'])
```

Comment:
Yeah I was changing the double to single quotes and then figured I'd make it shorter too. Singleton tuples are no fun.